dali/pipeline/executor/pipelined_executor.h

```diff
@@ -84,7 +84,7 @@ void PipelinedExecutorImpl<WorkspacePolicy, QueuePolicy>::Build(OpGraph *graph,
 }
 
 template <typename WorkspacePolicy, typename QueuePolicy>
-void PipelinedExecutorImpl<WorkspacePolicy, QueuePolicy>::SetupOutputInfo(const OpGraph &graph) {
+void PipelinedExecutorImpl<WorkspacePolicy, QueuePolicy>::SetupOutputInfo(OpGraph &graph) {
```

Comment:
Is there a particular reason to remove `const` here (and above)?

Reply:
SetupMakeContiguousPassThrough mutates the graph a bit.